sales/hubspot.md

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
I'd move this page to `marketing`. As most of the doc is about growing from contact to sales qualified leads